contracts/root/RootERC20Predicate.sol

```diff
@@ -108,6 +110,13 @@ contract RootERC20Predicate is Initializable, IRootERC20Predicate {
         return childToken;
     }
 
+    /**
+     * @inheritdoc IRootERC20Predicate
+     */
+    function nativeTokenRoot() external view returns (address) {
+        return _nativeTokenRootAddress;
+    }
+
```

Comment:
Sorry for my ignorance, but is it possible then to expose field declaration to the interface, because I thought it is better to provide interface (`IRootERC20Predicate`) rather than the implementation (`RootERC20Predicate`) to the `CustomSupernetManager`?

Reply:
No prob'. You don't need to change the interface at all, Solidity compiler will point it to the `public` variable as long as it has the same name as the function in the interface.

Reply:
Fixed https://github.com/0xPolygon/core-contracts/pull/411/commits/96f761f4880fac0381dfc56a47a7f05753546ab4

Reply:
Let me take a closer look and I'll update you.

Reply:
See review.